packages/perseus/src/widgets/radio/multiple-choice-component.new.tsx

```diff
@@ -249,34 +187,13 @@ const BaseRadio = ({
                             });
                         }
 
-                        const nextChoice = choices[i + 1];
-                        const nextChoiceHighlighted =
-                            nextChoice?.highlighted || false;
-
-                        const aphroditeClassName = (checked: boolean) => {
-                            // Whether or not to show correctness borders
-                            // for this choice and the next choice.
-                            return css(
-                                styles.item,
-                                styles.responsiveItem,
-                                checked && styles.selectedItem,
-                                checked &&
-                                    choice.highlighted &&
-                                    styles.aboveBackdrop,
-                                checked &&
-                                    choice.highlighted &&
-                                    apiOptions.isMobile &&
-                                    styles.aboveBackdropMobile,
-                                nextChoiceHighlighted &&
-                                    apiOptions.isMobile &&
-                                    styles.nextHighlighted,
-                            );
-                        };
-
-                        // HACK(abdulrahman): Preloads the selection-state
-                        // css because of a bug that causes iOS to lag
-                        // when selecting the button for the first time.
-                        aphroditeClassName(true);
+                        // Whether or not to show correctness borders
+                        // for this choice and the next choice.
```

Comment:
Just for my own understanding, what is a correctness border?

Reply:
No idea, this comment is an artifact from before. Perhaps related to highlighted. I'll remove it.